### ISSUE TITLE
Highline deprecation warnings make appliance_console difficult to use

### ISSUE THREAD

**Issue by agrare** [ManageIQ/manageiq-appliance_console#231]
Highline 1.6.21 with ruby 3.1 prints a number of deprecation warnings to the screen which causes the console to be difficult to use

```
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing safe_level with the 2nd argument of ERB.new is deprecated. Do not use it, and specify other arguments as keyword arguments.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing trim_mode with the 3rd argument of ERB.new is deprecated. Use keyword argument like ERB.new(str, trim_mode: ...) instead.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing safe_level with the 2nd argument of ERB.new is deprecated. Do not use it, and specify other arguments as keyword arguments.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing trim_mode with the 3rd argument of ERB.new is deprecated. Use keyword argument like ERB.new(str, trim_mode: ...) instead.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing safe_level with the 2nd argument of ERB.new is deprecated. Do not use it, and specify other arguments as keyword arguments.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing trim_mode with the 3rd argument of ERB.new is deprecated. Use keyword argument like ERB.new(str, trim_mode: ...) instead.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing safe_level with the 2nd argument of ERB.new is deprecated. Do not use it, and specify other arguments as keyword arguments.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing trim_mode with the 3rd argument of ERB.new is deprecated. Use keyword argument like ERB.new(str, trim_mode: ...) instead.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing safe_level with the 2nd argument of ERB.new is deprecated. Do not use it, and specify other arguments as keyword arguments.
/opt/manageiq/manageiq-gemset/gems/highline-1.6.21/lib/highline.rb:445: warning: Passing trim_mode with the 3rd argument of ERB.new is deprecated. Use keyword argument like ERB.new(str, trim_mode: ...) instead.
1) Configure Network
2) Create Database Backup
3) Create Database Dump
4) Restore Database From Backup
5) Configure Application
6) Configure Database Replication
7) Logfile Configuration
8) Control Application Database Failover Monitor
9) Configure External Authentication (httpd)
10) Update External Authentication Options
11) Generate Custom Encryption Key
12) Stop EVM Server Processes
13) Start EVM Server Processes
14) Restart Appliance
15) Shut Down Appliance
16) Summary Information
17) Quit
```

**Comment by Fryguy** [comment]
I thought we upgraded to 2.1?  https://github.com/ManageIQ/manageiq-appliance_console/pull/201

**Comment by agrare** [comment]
Oh maybe that bump just hasn't made it to a release yet

**Comment by Fryguy** [comment]
We have a ton of changes on master, but 8.0 was only released October 21, 2022.  I know we had started to go to switching to 8.0, but then backed out of the kafka changes.  However, when we enabled kafka I thought we'd have released another 8.0.  Note that  7.2.2 was released on June 30, 2023.

It feels like we missed something?  @nasark @bdunne @jrafanie ?

**Comment by nasark** [comment]
@Fryguy It looks like another 8.0 was not released https://github.com/ManageIQ/manageiq-appliance_console/tree/v8.0.0, but we switched the Gemfile in core to start using it. We're essentially using a version of appliance console from Oct 2022...?

**Comment by jrafanie** [comment]
yeah, I thought we manually backported things on 7.2 branch because the kafka on 8.0 was giving us problems when kafka was removed from the release.  Maybe we should review the changes and decide what we want to do.  I can see bringing more back to 7.2 if we're not ready for the 8.0 changes or we can just do it and go with 8.0.

**Comment by Fryguy** [comment]
We're already on 8.0 on master and quinteros, so im not sure why we haven't cut newer versions

**Comment by Fryguy** [comment]
I released v8.1.0 with the changes from master.  @agrare If you're still seeing this issue, please reopen.

**Comment by agrare** [comment]
Confirmed this is fixed with appliance_console v8.1.0
